jsfmt.js

```diff
@@ -32,3 +34,19 @@ stdin(function(data) {
     }
 });
 
+function getJsfmt(localJsfmt) {
+    var jsfmt;
+    var loader = path.join(localJsfmt, '__sublime-load-jsfmt.js');
```

Comment:
The assignment should be done inside the if statement as `path.join` will throw and error if the arguments are not strings.